assets/helpers/abtest.js

```diff
@@ -37,17 +37,29 @@ type Test = {
 };
 
 
+type OphanABEvent = {
+  variantName: string,
+  complete: boolean,
+  campaignCodes?: string[],
+};
+
+
+type OphanABPayload = {
+  [TestId]: OphanABEvent,
```

Comment:
It means that the name of the attribute will be type `TestId` and its value will be `OphanABEvent`.